src/view/newtListMenu.cpp

```diff
@@ -6,3 +6,76 @@
 // This file is empty since the method is now a template
 
 #include <cloysterhpc/view/newt.h>
+#include <newt.h>
+#include <ranges>
+
+static std::vector<std::string> retrieveListSelectedItems(newtComponent list)
+{
+    int selectCount = 0;
+    char** selectItems
+        = reinterpret_cast<char**>(newtListboxGetSelection(list, &selectCount));
+    auto temp = std::vector(selectItems, selectItems + selectCount)
+        | std::views::transform([](char* c) { return std::string { c }; });
+    std::vector<std::string> ret(std::begin(temp), std::end(temp));
+    return ret;
+}
+
+std::pair<int, std::vector<std::string>> Newt::multipleSelectionMenu(
+    const char* title, const char* message, const char* help,
+    std::vector<std::tuple<std::string, std::string, bool>> items)
+{
+    auto* form = newtForm(nullptr, nullptr, 0);
+    newtFormSetWidth(form, m_suggestedWidth);
+
+    auto* label = newtLabel(1, 1, message);
+    auto stitle = std::string { title };
```

Comment:
Don't get it. Is this an implicit type conversion?

Reply:
Yes

The compiler can deduce the type anyway; and since this label does not leave the function, the *exact* type name isn't too important

Reply:
Alright. Just use title directly no? In the end is another string. However if the plan is to store a `const char*` well... just use `title.c_str()` directly.